Source/Core/VideoBackends/Vulkan/VKGfx.cpp

```diff
@@ -250,18 +252,18 @@ void VKGfx::BindBackbuffer(const ClearColor& clear_color)
     {
       // The present keeps returning exclusive mode lost unless we re-create the swap chain.
       INFO_LOG_FMT(VIDEO, "Lost exclusive fullscreen.");
-      m_swap_chain->RecreateSwapChain();
+      m_swap_chain->RecreateSwapChain(m_backend->backend_info);
```

Comment:
You use `GetBackendInfo()` in some places and `m_backend->backend_info` in others.  While they are the same thing, should you use one for consistency?

Reply:
`GetBackendInfo()` returns a const reference to `backend_info`

The current implementation of `RecreateSwapchain` needs to modify `backend_info`. Should it be modifying backend_info? Probably not. But until then... it needs a non-const reference.